resources.go

```diff
@@ -1210,6 +1221,9 @@ func Provider() tfbridge.ProviderInfo {
 			// NetApp
 			"azurerm_netapp_account": {Tok: azureResource(azureNetapp, "Account")},
 			"azurerm_netapp_pool":    {Tok: azureResource(azureNetapp, "Pool")},
+
+			//AppConfiguration
+			"azurerm_app_configuration": {Tok: azureResource(azureAppConfiguration, "ConfigurationStore")},
```

Comment:
I'm happy with `ConfigurationStore`. `Store` could read more natural in TypeScript, but is more explicit in namespace-less .NET code.

Reply:
👍